### PR TITLE
logging options are wrong

### PR DESCRIPTION
**Pull request by jacarey**
One more time with feeling!

----

### Checklist (never delete this)

Never delete this, it is our record that procedure was followed. If you find that for whatever reason one of the checklist points doesn't apply to your PR, you can leave it unchecked but please add an explanation below.

- [ ] **Submitter**: Include the JIRA issue number in the PR description
- [ ] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [ ] **Submitter**: Ensure that you have added or updated tests
- [ ] **Submitter**: If you're adding/switching libraries or otherwise changing the design, update the [design documentation](https://broadinstitute.atlassian.net/wiki/pages/viewpage.action?pageId=114531509).
- [ ] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * Add notes on what you've tested
* Review cycle:
  * Team may comment on PR at will
  * **Reviewer assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to reviewer** for further feedback
- [ ] **Submitter**: Verify all tests go green, including CI tests
- [ ] **Submitter**: Squash commits and merge to develop
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Move the JIRA issue to QA once this checklist is completed

### CODE REVIEW

File: clio-server/config/docker-compose.yml.ctmpl [hunk 1]
```diff
@@ -6,9 +6,9 @@
 app:
   image: broadinstitute/clio-server:{{env "DOCKER_TAG"}}
   logging:
```
Comment (danxmoran) [inline clio-server/config/docker-compose.yml.ctmpl]: I think you need to delete `logging` entirely and nest the two options directly under `app`, based on the cromwell layout and the last error message

**Review by danxmoran — CHANGES_REQUESTED** [review]

File: clio-server/config/docker-compose.yml.ctmpl (hunk 1 above)
Comment (jacarey) [reply clio-server/config/docker-compose.yml.ctmpl]: yup just repushed

File: clio-server/config/docker-compose.yml.ctmpl [hunk 2]
```diff
@@ -5,10 +5,9 @@
 {{with $env_file := env "CLIO_ENV_FILE"}}
 app:
   image: broadinstitute/clio-server:{{env "DOCKER_TAG"}}
-  logging:
-    driver: syslog
-    options:
-        tag: "clio-server"
+  log_driver: "syslog"
+  log_opt:
+      tag: "clio-server"
```
Comment (danxmoran) [inline clio-server/config/docker-compose.yml.ctmpl]: I think this has one-too-many indent compared to the rest of the file (4 spaces instead of 2)

**Review by danxmoran — APPROVED** [review]
One indentation nitpick, not sure if YAML will freak out over it

**Pull request merged**